sql/core/src/test/scala/org/apache/spark/sql/SubquerySuite.scala

```diff
@@ -1876,4 +1877,29 @@ class SubquerySuite extends QueryTest with SharedSparkSession with AdaptiveSpark
         "ReusedSubqueryExec should reuse an existing subquery")
     }
   }
+
+  test("SPARK-36280: Remove redundant aliases after RewritePredicateSubquery") {
+    sql("CREATE TABLE t1 USING parquet AS SELECT id AS a, id AS b, id AS c FROM range(10)")
```

Comment:
nit: wrap with `withTable`

Reply:
Fixed it. Sorry. I don't know why I forgot.